pkg/systemd/services.html

```diff
@@ -122,6 +122,23 @@
             {{/TemplateDescription}}
           </table>
         </div>
+        {{#NotMetConditions.length}}
+        <div class="list-group-item">
+          <table>
+            <tr>
+              <td>
+                <span>condition failed</span>
```

Comment:
This needs to be translatable using a ```<span translate>``` attribute. And shouldn't ```condition``` be capitalized?